.github/workflows/release.yaml

```diff
@@ -162,6 +162,20 @@ jobs:
         artifacts: ${{ env.ASSET }}
         artifactContentType: application/octet-stream
 
+  finalize-release:
+    name: finalize-release
+    needs: [create-release, build-release]
```

Comment:
```suggestion
    needs: [build-release]
```

Reply:
this uses `needs.create-release.outputs.ironhide_version`, so I'm assuming that means it needs both